docs/server-request.md

```diff
@@ -17,20 +17,38 @@ A low-level action that allows for any kind of HTTP request to be performed.
 - Read files or binary streams
 - Call APIs from same app system - use the [Call Local API](/docs/Actions/CallLocalApi.md) action instead
 
-## `Related Actions`
+## `Related actions`
 
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
 
-## `Parameter Reference`
+## `Parameter reference`
 
-| Parameter     | Description                           | Supports Variables |
+| Parameter     | Description                           | Supports variables |
 |---------------|---------------------------------------|--------------------|
 | URL           | Absolute URL where the resource to be invoked lives. | Yes |
-| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD. | No |
+| Enforce SSL   | This will for ce request to be sent to a secure version of the URL.(https instead of http)| No |
+| Timeout       | The amount of time in which the response must come or the request fails. | Yes |
```

Comment:
What is the default?
What happens with the rest of the actions if this Timeout is 1h?
How can I make the process continue without waiting for the response and have some other actions executed when the request completes?

Reply:
100 seconds. https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.timeout?view=netcore-3.1
Depends. The rest of the actions will continue execution when and if the server replies under 1h, otherwise an exception will be thrown.
There is no way to leverage that in the current form of the action code, as far as I am aware of. That would imply leveraging async execution.

Reply:
Then let's document the default and link to the async action as a suggestion for the case where the process doesn't have to wait for the request to finish.

Reply:
What async action are you referring to? What I meant is that our actions can't execute in an async fashion, there is no way around this limitation.